src/Murder.Editor/EditorScene_Shortcuts.cs

```diff
@@ -0,0 +1,263 @@
+        {
+            _f5Lock = false;
+        }
+        
+        if (Game.Input.Shortcut(Keys.W, _leftOsActionModifier) ||
+            Game.Input.Shortcut(Keys.W, _rightOsActionModifier))
+        {
+            CloseTab(_selectedAssets[_selectedTab]);
+        }
+
+        if (Game.Input.Shortcut(Keys.F, _leftOsActionModifier) || 
+            Game.Input.Shortcut(Keys.F, _rightOsActionModifier))
+        {
+            _focusOnFind = true;
+        }
+    }
+
+    private static bool DrawShortcut(Shortcut shortcut)
+    {
+        if (shortcut is ToggleShortcut toggleShortcut)
+        {
+            return ImGui.MenuItem(shortcut.Name, shortcut.Chord.ToString(), ref toggleShortcut.Checked);
+        }
+        
+        return ImGui.MenuItem(shortcut.Name, shortcut.Chord.ToString());
+    }
+
+    private void RefreshEditorWindow()
+    {
+        Architect.Instance.SaveWindowPosition();
+        Architect.Instance.RefreshWindow();
+    }
+
+    private void RunDiagnostics()
+    {
+        if (_selectedTab == Guid.Empty || !_selectedAssets.TryGetValue(_selectedTab, out GameAsset? asset))
+        {
+            GameLogger.Warning("An asset must be opened in order to run diagnostics.");
+        }
+        else
+        {
+            CustomEditorInstance? instance = GetOrCreateAssetEditor(asset);
+            if (instance?.Editor.RunDiagnostics() ?? true)
+            {
+                GameLogger.Log($"\uf00c Successfully ran diagnostics on {asset.Name}.");
+            }
+            else
+            {
+                GameLogger.Log($"\uf00d Issue found while running diagnostics on {asset.Name}.");
+            }
+        }
+    }
+
+    private void ReloadAtlasWithChangesToggled(bool value)
+    {
+        Architect.EditorSettings.OnlyReloadAtlasWithChanges = value;
+    }
+
+    private static void ReloadContentAndAtlas()
+    {
+        _ = Architect.EditorData.ReloadSprites();
+        AssetsFilter.RefreshCache();
+    }
+
+    private static void ReloadShaders()
+    {
+        Architect.Instance.ReloadShaders();
+    }
+
+    private static void ReloadSounds()
+    {
+        _ = Game.Data.LoadSounds(reload: true);
+    }
+
+    private static void SaveAllAssets()
+    {
+        Architect.EditorData.SaveAllAssets();
+    }
+
+    private static void BakeAsepriteGuids()
+    {
+        AsepriteServices.BakeAllAsepriteFileGuid();
+    }
+
+    private void StartGame()
+    {
+        SaveEditorState();
+        Architect.Instance.QueueStartPlayingGame(false);
+    }
+
+    private static void ToggleGameLogger()
+    {
+        GameLogger.GetOrCreateInstance().ToggleDebugWindow();
+    }
+
+    private enum ShortcutGroup
+    {
+        Game,
+        Assets,
+        View,
+        Reload,
+        Tools
+    }
+
+    private abstract record Shortcut(string Name, Chord Chord)
+    {
+        public abstract void Execute();
+    }
+
+    private sealed record ActionShortcut(
```

Comment:
maybe we can make some of these `protected` if people want to extend their own stuff/specific for the game (afterwards)